### DISCUSSION TITLE
Push Kit v6.7.0.300 added the "install HMS Core" prompt again?

### DISCUSSION THREAD

**Discussion by nunoarruda** [HMS-Core/hms-cordova-plugin#110] [Q&A] [answered]
Push Kit [v6.3.0.304](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/android-app-version-0000001074227861#section1647171217715) deleted the "install HMS Core" prompt because of #81:

<img width="537" alt="Screenshot 2022-10-03 at 15 10 26" src="https://user-images.githubusercontent.com/1189149/193598777-f60dcd5c-270a-4d25-937f-862d438d924b.png">

But it looks like it was added back again in [v6.7.0.300](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/android-app-version-0000001074227861#section988583822719):

<img width="860" alt="Screenshot 2022-10-03 at 15 08 10" src="https://user-images.githubusercontent.com/1189149/193598931-23c3b2b1-1b45-4139-a257-abb3fc3991bf.png">

Will this not cause the issues reported in #81 all over again?

**Comment by xuxiapu** [comment, marked as answer]
The installation capability has not been added to the push SDK. It is a separate SDK. There are integration instructions in the [document](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/android-app-version-0000001074227861#section988583822719). Please refer to this [link](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/android-integrating-sdk-0000001050040084#EN-US_TOPIC_0000001087049212__li1778761702119).
![image](https://user-images.githubusercontent.com/114389983/194685739-0aa24613-5b7a-436e-8841-094557b84f79.png)
